app/src/main/java/ir/ac/shirazu/softwareproject/server_api/Meal/MyKit.java

```diff
@@ -245,19 +243,28 @@ public Student studentLogin(String userName, String password) {
             String userInformation = sendPostRequest(tokenToJson(userToken), studentInfoURL);
 
 
-
             //JSON Informations
             HashMap<String, JSONObject> list = new HashMap<>();
             totalInfoToJSONs(userInformation, list);
 
+
             //Fill Student personal Data
             fillStudentPersonalData(newStudent, list.get("student"));
 
 
             //Fill MealInfo
             fillStudentMealInfo(newStudent, list.get("coupons"));
+
+
+            //Fill Self FoodInfo
+            fillSelfFoodInfo(list.get("self_data"));
+
+
+            //Logging out
             sendPostRequest(tokenToJson(userToken),logoutURL);
```

Comment:
@aminsoheyli this is the log out function that maaz has used to log out.

Reply:
@aminsoheyli did you delete that protected function ?
I suggest not to delete that untill i find out wether it has any negative impact on the project or not.

Reply:
Yeah, I seen it.

Reply:
> @aminsoheyli did you delete that protected function ?
> I suggest not to delete that untill i find out wether it has any negative impact on the project or not.

@hosseindehghanipour1998 No, I don't manipulate this code until your last commit.

Reply:
Has it been completed?